plugins/config-file.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: config-file               
+spec:
+  version: "0.0.6"      
+  platforms:
+  - selector:             
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/Modulus/kubectl-cfg/archive/0.0.6.zip
+    sha256: "41952b7105e2a9f5f1b3fec306bc59c8e972578f9b5d543defeb970c5b5876d5"
+    files:
+    - from: "/kubectl-config-file*/*.sh" 
+      to: "."              
+    bin: "./kubectl-config-file.sh"  
+  shortDescription: List config files, and links chosen config file to ~/.kube/config without messing up your files.
+  homepage: https://github.com/Modulus/kubectl-cfg 
+  caveats: |
+    This plugin needs the following programs:
+    * ls(1)
```

Comment:
Since you're installing on darwin/linux only, I'd say you can skip listing all of these tools. They're all probably preinstalled?

However, make sure in your plugin you check if they exist first in the startup.

Reply:
I have added a plugin check to the shell program